ide/static/js/content.js

```diff
@@ -1072,6 +1101,13 @@ class Content extends React.Component {
           // while loading a model ensure paramete intialisation
           // for UI show/hide is not executed, it leads to inconsistent
           // data which cannot be used further
+          if (response.public_sharing == false) {
+            is_shared = false;
```

Comment:
If you are not changing value before this step then why do you have to set it to false again as the default value is already false

Reply:
Okay, I’ll make the change, thanks for pointing out!